roles/openshift_node/tasks/config.yml

```diff
@@ -40,15 +41,27 @@
 - name: Fetch bootstrap ignition file locally
   uri:
     url: "{{ openshift_node_bootstrap_endpoint }}"
-    dest: "{{ tempfile.path }}/bootstrap.ign"
+    dest: "{{ temp_dir.path }}/bootstrap.ign"
     validate_certs: false
 
-- name: Copy pull secret in the directory
+- name: Get cluster pull-secret
+  command: >
+    oc get secret pull-secret
```

Comment:
I could have missed it but what namespace is this secret in?

Reply:
Added namespace